### PR TITLE
Task 795 navbar brand v c

### PR DESCRIPTION
**Pull request by dancingolav**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java [hunk 1]
```diff
@@ -28,73 +27,69 @@ public void before() {
...
-                    imgFromNavbar.is().attr("src", containsString(imgPath))
-                            .tag("img");
-                    imgFromNavbar.unhighlight();
+    @Test (dataProvider = "navbarBrandsWithLink")
+    public void  checkNavbarLink(UIElement brandAsLink) {
+        brandAsLink.is().core().hasAttr("href");
+        brandAsLink.highlight("blue");
+        brandAsLink.unhighlight();
+        brandAsLink.click();
+        int winNumber = WindowsManager.windowsCount();
```
Comment (AUlasen) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java]: Method `WindowsManager.windowsCount()` return current count of browser tabs, it doesn't wait anything.  So, when you call it right after the click, result will be unpredictable. In such case it is better to write:

```
int winNumber = WindowsManager.windowsCount();
brandAsLink.click();
WindowsManager.switchToWindow(winNumber + 1);
```
Method `WindowsManager.switchToWindow(winNumber + 1)` will wait, while tab `winNumber + 1` will really apear.

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java [hunk 2]
```diff
@@ -28,73 +27,69 @@ public void before() {
...
+
+    @Test(dataProvider = "navbarBrandsWithImage")
+    public void checkNavbarImage(UIElement brandWithImage) {
+        UIElement imgFromNavbar = brandWithImage.childs().get(1);
+        imgFromNavbar.highlight("blue");
+        imgFromNavbar.is().core().tag("img").attr("src", containsString(imgPath));
+        imgFromNavbar.unhighlight();
+        imgFromNavbar.click();
+        int winNumber = WindowsManager.windowsCount();
+        WindowsManager.switchToWindow(winNumber);
```
Comment (AUlasen) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java]: The same.

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java (hunk 1 above)
Comment (dancingolav) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java]: ok

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java (hunk 2 above)
Comment (dancingolav) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarBrandTests.java]: ok

**Pull request merged**